src/Microsoft.DotNet.ProjectModel/Resolution/LibraryManager.cs

```diff
@@ -157,7 +157,7 @@ public IList<DiagnosticMessage> GetAllDiagnostics()
 
                         if (item.Library != library && !versionRange.Satisfies(library.Identity.Version))
                         {
-                            var message = $"Dependency conflict. {item.Library.Identity} expected {FormatLibraryRange(item.Dependency)} but got {library.Identity.Version}";
+                            var message = $"Dependency conflict. {item.Library.Identity} expected {FormatLibraryRange(item.Dependency)} but received {library.Identity.Version}";
```

Comment:
Ok, so grammar nitpicking begins now :).

Perhaps `found` is a more appropriate word here? After all, the resolution process is going out and finding dependencies, it's not being given them (and thus not receiving them).